### PR TITLE
Bugfix file permissions, remove unneccesary login info and readme update

### PR DESCRIPTION
**Pull request by hlogmans**
The file permissions on the generated server config file needs to be world readable otherwise it is skipped (at least on Ubuntu 14 under Apache 2.4.7).

De user and password info must be empty when using cookie authentication.

Added some quickstart in the README.

Updated the version number.

### CODE REVIEW

File: providers/db.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	        :pma_db => new_resource.pma_database,
 	        :auth_type => new_resource.auth_type
 		})
-		mode 00640
+		mode 00644
```
Comment (priestjim) [inline providers/db.rb]: Why? This file contains passwords and should not be world readable!

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.0.7
+1.0.8
```
Comment (priestjim) [inline VERSION]: Please do not update the cookbook's release version as there might be more changes coming up to bundle into a release version bump

File: README.md [hunk 3]
```diff
@@ -106,7 +106,58 @@ To define a database config you can use the phpmyadmin_db LWRP such as:
 		hide_dbs %w{ information_schema mysql phpmyadmin performance_schema }
 	end
 
-This will create a file in /opt/phpmyadmin/conf.d/test_db.inc.php and will be automatically included when you display the PMA page.
```
Comment (priestjim) [inline README.md]: Why was this line removed?

File: providers/db.rb (hunk 1 above)
Comment (hlogmans) [inline providers/db.rb]: It didn't work without it, it was the reason why I could not login into PMA (this file was not readable), I might need to review the PMA owner and group, so it IS accessible for a more limited audience.

File: README.md (hunk 3 above)
Comment (hlogmans) [inline README.md]: Hmmm, probably overlooked when creating the PR...

File: providers/db.rb (hunk 1 above)
Comment (priestjim) [inline providers/db.rb]: The reason the permissions were set in that way is that php-fpm does not need "other" permissions to read that file but in your case, using Apache, it does (apache does not run with the file owner's permission but with www-data/httpd). You should add a check for node['phpmyadmin']['fpm'] there to set the appropriate permissions.

**Comment by priestjim** [comment]
Closing due to lack of feedback.
